tests/features/language-selection.feature

```diff
@@ -23,6 +26,9 @@ Feature: Language selection
     When I click "français"
     Then the url should match "/fr"
 
+    When I click "português"
+    Then the url should match "/pt-pt"
```

Comment:
URL should still be `/pt` the scope of the issue is to just change the language code, not the URL prefix.